packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.base.tsx

```diff
@@ -280,7 +280,7 @@ export class GroupedListBase extends BaseComponent<IGroupedListProps, IGroupedLi
     const groups = this.state.groups;
     const pageGroup = groups && groups[itemIndex];
     return {
-      key: pageGroup && pageGroup.name
+      key: pageGroup && pageGroup.key
```

Comment:
Is key exposed anyhow? The primary reason I was thinking of leaving name in as a fallback was for backwards compatibility.

Reply:
Not that I could find. I made the determination that `key` was not part of our public API contract and therefore did not require the fallback. `key` is required so it is _always_ present so we should be OK with this change. Sorry missed the comment but wanted to follow-up.